File: Python/595A-vitalyandnight.py
n, m = map(int, input().split())
answer = 0
floors = []
for i in range(n):
    floor = list(map(int, input().split()))
    floors.append(floor)
for i in range(n):
    for j in range(0, 2 * m, 2):
        answer += (1 if floors[i][j] == 1 or floors[i][j + 1] == 1 else 0)
print(answer)

File: C++/A/595A-vitalyandnight.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n, m, answer = 0;
    cin >> n >> m;
    vector<vector<int>> floors;
    for (int i = 0; i < n; i++)
    {
        vector<int> floor;
        for (int j = 0; j < 2 * m; j++)
        {
            char x;
            cin >> x;
            floor.push_back(x);
        }
        floors.push_back(floor);
    }
    for (int i = 0; i < n; i++)
        for (int j = 0; j < 2 * m; j += 2)
            answer += ((floors[i][j] == '1' || floors[i][j + 1] == '1') ? 1 : 0);
    cout << answer;
}